tests/python/unittest/test_meta_schedule_schedule_rule_mlt.py

```diff
@@ -798,4 +885,5 @@ def max_pool_blocked_compute(height, width, channel):
 
 
 if __name__ == "__main__":
-    tvm.testing.main()
+    # tvm.testing.main()
+    test_cache_read_specify_consumer()
```

Comment:
Please revert this change :-)

Reply:
Oh ooops! Will do in a quick follow-up PR!

Reply:
Okay thanks